cases/drp-00-05.tex

```diff
@@ -26,32 +34,149 @@ \subsubsection{Intercase dependencies}
 
 \subsubsection{Environmental needs}
 
-\paragraph{Hardware}
+\paragraph{Hardware}\label{sec:hardware}
+
+This test case shall be executed on a system at NCSA.
 
-\begin{note}
-Data Facility to provide content.
-\end{note}
+For LDM-503-2, this NCSA system will be the LSST Verification Cluster (LSST-VC),
+with read/write access to the GPFS shared filesystem and a single node Oracle database.
 
-\paragraph{Software}
+\paragraph{Software}\label{sec:software}
+The LSST DM Software Stack and the DESDM Framework packages will be pre-installed.
```

Comment:
For LDM-503-2 it's a separate installation.  In the future it may be a same installation.

Reply:
Because the software is spread across 3 different sources, the installation process is currently an interesting manual process.    As @hsinfang stated, if/when the DESDM software base has been chosen to become part of the LSST software stack, it would fall under DRP-00-00.  This decision is slated for late 2018/very early 2019 AFAIK.   We had planned on writing up this manual process post-Nov after having more experience installing on machines from scratch.    How important is it that this temporary installation process is written down for this version of the document?